Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "fuel-debugger"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+members = ["client"]
+
+[dependencies]
+fuel-types = { version = "0.1", features = ["serde-types"] }
+fuel-vm = { version = "0.5.1", features = ["serde-types", "debug"] }
```

Comment:
Should you use the latest version of these deps? Also, use major.minor instead of major.minor.patch unless there's a specific reason not to.

Reply:
I updated `fuel-types` to latest version. For `fuel-vm`, version `0.5.0` will not work, and specifying `0.5.1` allows cargo to use any `0.5` version expect `0.5.0`.

Reply:
Can you add a comment to that effect here?

Reply:
Done